openedx/adg/lms/applications/test/test_views.py

```diff
@@ -0,0 +1,44 @@
+"""
+All tests for applications views
+"""
+from django.test import Client, TestCase
+from django.test.client import RequestFactory
+
+from openedx.adg.lms.applications.test.factories import ExtendedUserProfileFactory
+from openedx.adg.lms.applications.views import ContactInformationView
+from student.tests.factories import UserFactory
+
+
+class ContactInformationViewTest(TestCase):
+    """
+    Test cases for the ContactInformationView
+    """
+
+    def setUp(self):
+        """
+        Create a new user and logged in to access the view
+        """
+        self.client = Client()
+        self.user = UserFactory()
+
+        extended_profile = ExtendedUserProfileFactory()
+        extended_profile.user = self.user
+        extended_profile.save()
+
+        self.client.login(username=self.user.username, pasword=self.user.password)
```

Comment:
Are we using this client anywhere? If not, please remove it.
I guess we are only using `RequestFactory` to make requests.